stubs/django-import-export/import_export/tmp_storages.pyi

```diff
@@ -1,30 +1,34 @@
+from abc import abstractmethod
 from typing import IO, Any, ClassVar
 
 class BaseStorage:
     name: str | None
     read_mode: str
     encoding: str | None
     def __init__(self, *, name: str | None = None, read_mode: str = "", encoding: str | None = None) -> None: ...
+    @abstractmethod
     def save(self, data: Any) -> None: ...
-    def read(self) -> None: ...
+    @abstractmethod
+    def read(self) -> Any: ...
```

Comment:
Could you add a comment why `Any` is used here, as we require that now per our policy? Same with the `read()` return types below.

Reply:
Done!